src/app/organizations/collection/collection.component.html

```diff
@@ -30,6 +30,10 @@
           You can view your pending organization requests on the requests tab of the <a [routerLink]="('/accounts')" style="text-decoration: underline">accounts page</a>.
         </ng-template>
       </mat-card>
+
+      <mat-card fxFlex class="my-3 alert alert-info" *ngIf="organization?.status === 'REJECTED'">
```

Comment:
I don't get the whole context, but assuming this is a public page (if it's not, ignore the rest of this):

Is this for the case where an org was approved then rejected?

If not, I think rejected orgs should essentially be treated as non-existent to non-members.

If yes, it's trickier because of the whole deletion/archiving thing we've been discussing.

Reply:
So this should only show if the user is a curator/admin or has a membership in the organisation. It should not appear to anyone else (public).